css/rnx.css

```
@@ -0,0 +1,366 @@
+
+  /* lines */
+  --rnx-border-color: #d3dae1;
+  --rnx-border-color-light: #e4e9ee;
+  --rnx-border-width: 1px;
+  --rnx-border-radius: 9px;
+  --rnx-border-radius-sm: 6px;
+  --rnx-border-radius-lg: 12px;
+
+  /* type */
+  --rnx-font-family: "Public Sans", ui-sans-serif, system-ui, -apple-system, "Segoe UI", sans-serif;
+  --rnx-font-family-display: Archivo, ui-sans-serif, system-ui, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Archivo" to be "archivo" (value-keyword-case)

(value-keyword-case)